### ISSUE TITLE
Allow search by schemas in cmd + P

### ISSUE THREAD

**Issue by lcjury** [TablePlus/TablePlus#1184]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
2. Which TablePlus build number are you using: v 2.3 (222)

3. The steps to reproduce this issue:
I have some kind of multi-tenancy on my develop machine, and when I search a table using cmd + P the tables from all the schemas are listed

![image](https://user-images.githubusercontent.com/1194606/57857874-00a7cf80-77be-11e9-8572-178e81e9f6d9.png)

But I'm not allowed to say, for example, "schema public", that would be nice.

**Comment by huyphams** [comment]
Thank you @lcjury for the insight

**Comment by lcjury** [comment]
Thanks to you guys, tableplus is awesome :)

**Comment by brianjsw** [comment]
Yes I just encountered this same problem today.  We are a multi-tenant provider and we have hundreds of schemas in the same database with the same table names.  This would be wonderful if we could type, say schema.tab___ and have it jump and find the table.  Even sorting the results by schema would be an improvement.
